src/Sentiment/index.js

```diff
@@ -0,0 +1,95 @@
+import * as tf from '@tensorflow/tfjs';
+import callCallback from '../utils/callcallback';
+/**
+* Initializes the Sentiment demo.
+*/
+
+const OOV_CHAR = 2;
+const PAD_CHAR = 0;
+
+function padSequences(sequences, maxLen, padding = 'pre', truncating = 'pre', value = PAD_CHAR) {
```

Comment:
Should this go in [utils](https://github.com/ml5js/ml5-library/tree/development/src/utils)?

Reply:
let's discuss this.